calculator/script.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var strCalc = "";
+  clearCalc();
+  document.querySelector(".btn-c").onclick = function () {
```

Comment:
bad practice, better to use addEventListener and for parent element, then check event.target